SU-HSDO/suhumsci
# feat(STN-306): background color for Row Paragraph type

# READY FOR REVIEW

## Summary
Adds a new field for the paragraph style on the Row paragraph type.

**Browser testing**

## Steps to Test
1. In humsci_basic, run `npm run test`
2. If you have not synced the latest from econ, I would recommend doing that now, on my branch: `lando blt drupal:sync --site=economics --sync-files --partial`.
3. If you have pulled from econ recently, then you can just import the new database changes: `lando drush @economics.local config-import --partial`
4. After logging into the site, go to a basic page, and either edit or add a new Paragraph "Row" type. Make sure to check for the existence of the new select field called "Paragraph Style". Select a paragraph style and then save. Check your page to see if the changes were applied.
5. Check both limited width and full width options. Also make sure to check a row on a page with a sidebar (both options should become limited width).

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
